tests/testthat/test-palaeorotate.R

```diff
@@ -1,32 +1,24 @@
-test_that("palaeorotate() works", {
+test_that("palaeorotate() point method works", {
+  test <- try(exp = GET(
+    paste0("https://gws.gplates.org/reconstruct/reconstruct_points/",
+           "?points=95,54,142,-33&time=140&model=SETON2012")
+  ), silent = TRUE)
+  test <- grep("Error", test)
+  if (test == 1) {
+    skip("GPlates API not available.")
+  }
```

Comment:
Would you be able to just use `skip_if_offline()` instead? https://testthat.r-lib.org/reference/skip.html